pkg/distro/fedora/package_sets.go

```diff
@@ -680,6 +680,7 @@ func iotSimplifiedInstallerPackageSet(t *imageType) rpmmd.PackageSet {
 			"sudo",
 			"traceroute",
 			"util-linux",
+			"util-linux-core",
```

Comment:
Maybe (probably?) I'm missing something, but isn't `util-linux-core` a hard depedency of `util-linux`, i.e. don't we always install this automatically anyway?

Reply:
Damn, no, you are not missing anything, `util-linux-core` is a hard dependency of `util-linux` so it must be installed anyway. I think the problem appeared when I was "playing" with dracut modules to see which of them where not required. I guess that in one of my attempts I removed `dracut-systemd` from the list which is the one installing `sulogin`

I will remove the commit from the PR, sorry